rcvSpec.yaml

```diff
@@ -0,0 +1,367 @@
+                $ref: '#/components/schemas/Poll'
+        '400':
+          description: Invalid PollID supplied
+          content: {}
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UnauthorizedError' 
+        '404':
+          description: Poll not found
+          content: {}
+
+components:
+  schemas:
+    Poll:
+      type: object
+      properties:
+        pollId:
+          type: integer
+          format: int64
+        rankedChoiceCount:
+          type: integer
+          format: int32
+        candidates:
+          type: array
+          items:
+            $ref: '#/components/schemas/Candidate'
+
+    PollDetails:
+      type: object
+      properties:
+        startDate:
+          type: string
+          example: '2021-04-28 01:45:00.0000 -05:00'
+        endDate:
+          type: string
+          example: '2021-05-01 12:00:00.0000 -05:00'
+        rankedChoiceCount:
+          type: integer
+          format: int32
+        candidates:
+          type: array
+          items:
+            $ref: '#/components/schemas/Candidate'
+
+    PollResults:
+      type: object
+      properties:
+        pollId:
+          type: integer
+          format: int64
+        totalEntries:
+          type: integer
+          format: int32
+        totalSteps:
+          type: integer
+          format: int32
+        winner:
+          type: string
+          example: Spiderman
+        steps:
+          type: array
+          items:
+            $ref: '#/components/schemas/ResultStep'
+
+    ResultStep:
+      type: object
+      properties:
+        stepId:
+          type: integer
+          format: int64
+        candidates:
+          type: array
+          items:
+            $ref: '#/components/schemas/CandidateResults'
+
+    Candidate:
+      type: object
+      properties:
+        name:
+          type: string
+          example: Spiderman
+
+    CandidateResults:
+      type: object
+      properties:
+        name:
+          type: string
+          example: Spiderman
+        votes:
+          type: array
+          items:
+            $ref: '#/components/schemas/VoteCount'
+
+    VoteCount:
+      type: object
+      properties:
+        choicePosition:
+          type: integer
+          format: int32
+        voteCount:
+          type: integer
+          format: int32
+
+    VoteInput:
+      type: object
+      properties:
+        choices:
+          type: array
+          items:
+            $ref: '#/components/schemas/Vote'
+
+    Vote:
+      type: object
+      properties:
+        choicePosition:
+          type: integer
+          format: int32
+        candidate:
+          $ref: '#/components/schemas/CandidateVote'
+
+    CandidateVote:
+      type: object
+      properties:
+        name: # TODO: Maybe we shouldn't key based off of string name, should we give each candidate a unique ID?

```

Comment:
In the meeting we decided we should have a unique id. I suggest ulid.

Threw in a TODO in our User Story Docs to decide whether we want to allow the same candidate name for multiple candidates in the same poll. (Decision doesn't effect this schema)